ReplayKitExample/ReplayKitExample/ReplayKitVideoSource.swift

```diff
@@ -31,11 +32,38 @@ class ReplayKitVideoSource: NSObject, TVIVideoCapturer {
     static let kYPlane = 0
     static let kUVPlane = 1
 
-    var lastTimestamp: CMTime?
     var downscaleBuffers: Bool = false
     var screencastUsage: Bool = false
     weak var captureConsumer: TVIVideoCaptureConsumer?
 
+    var lastTimestamp: CMTime?
+    var timerSource: DispatchSourceTimer?
+    var lastTransmitTimestamp: CMTime?
+    var retransmitTimer: Timer?
+    var retransmitLock = os_unfair_lock()
+
+    private var lastFrameStorage: TVIVideoFrame?
+
+    var lastFrame: TVIVideoFrame? {
+        get {
+            os_unfair_lock_lock(&retransmitLock)
+            let frame = lastFrameStorage
+            os_unfair_lock_unlock(&retransmitLock)
+            return frame
+        }
+        set {
+            os_unfair_lock_lock(&retransmitLock)
+            lastFrameStorage = lastFrame
+            os_unfair_lock_unlock(&retransmitLock)
+        }
+    }
```

Comment:
Did I miss some initialization? Crashing in `os_unfair_lock_lock`. This code path is disabled by default.